api/lib/ZnifferManager.ts

```diff
@@ -113,33 +132,8 @@ export default class ZnifferManager extends TypedEventEmitter<ZnifferManagerEven
 	private async init() {
 		try {
 			await this.zniffer.init()
-
-			this.zniffer.on('frame', (frame, rawData) => {
-				const socketFrame = this.parseFrame(frame, rawData)
-
-				this.socket.emit(socketEvents.znifferFrame, socketFrame)
-			})
-
-			this.zniffer.on('corrupted frame', (frame, rawData) => {
-				const socketFrame = this.parseFrame(frame, rawData)
-
-				this.socket.emit(socketEvents.znifferFrame, socketFrame)
-			})
-
-			this.zniffer.on('error', (error) => {
-				this.onError(error)
-			})
-
-			this.zniffer.on('ready', () => {
-				logger.info('Zniffer ready')
-			})
 		} catch (error) {
-			this.onError(error)
-
 			logger.info('Retrying in 5s...')
-
-			await this.close()
```

Comment:
This prevents reusing the Zniffer instance. Alternatively, constructing the Zniffer instance would have to be done in `this.init()` too.